docs/getting-started-with-micro/advanced-usage/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Advanced usage"
+sidebar_position: 2
+description: "How to configure Snowplow Micro and expose it to the world."
+---
+
+```mdx-code-block
+import {versions} from '@site/src/componentVersions';
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+## Adding custom Iglu resolver configuration
+
+There are two ways to add your own [schemas](/docs/understanding-tracking-design/understanding-schemas-and-validation/index.md) to Micro:
+* By loading your schema files directly into Micro (see [Basic usage](/docs/getting-started-with-micro/basic-usage/index.md#adding-custom-schemas))
+* By pointing Micro to your Iglu resolver
+
+Let’s explore the latter option. First, create your [Iglu resolver configuration](/docs/pipeline-components-and-applications/iglu/iglu-resolver/index.md), including your custom resolver. Place it in `iglu.json`.
+
+<details>
+<summary>Example</summary>
+
+```json reference title="iglu.json"
+https://github.com/snowplow-incubator/snowplow-micro/blob/master/example/iglu.json
+```
+
+</details>
```

Comment:
This example doesn't feel particularly useful given that this is specifically about adding a custom repository and you get iglucentral by default. 

Also it should probably say `..., including your custom repository.` instead of resolver.

Reply:
I’ve updated this to not include the example. The configuration link in the first sentence actually gives a much better example already! Besides, this is “advanced usage”, so people are expected to know what they are doing. I’ve covered adding a single custom registry the easy way in “basic usage”.